sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -14,10 +14,11 @@ package org.scalajs.sbtplugin
 
 import scala.annotation.tailrec
 
-import scala.concurrent._
+// sbt.Future collides, but only exists in 0.13 so cannot unimport easily.
+import scala.concurrent.{Future => scFuture, _}
```

Comment:
Isn't it sufficient to use a named import for
```scala
import scala.concurrent.Future
```
? Named imports are supposed to take precedent over wildcard imports.

Reply:
Ah, yes, good point.